load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -1645,3 +1644,14 @@ func (s *relaysSuite) populateWithKnownGateways() (gateways []*accountInfo) {
 
 	return gateways
 }
+
+// getTestTotalRelayCost computes the total relay cost for the test in uPOKT.
```

Comment:
#PUC

It uses some params internal to `s`, somehow leverages the variables provided.

I really have to read the code in the function to understand this.